MavenQuickStart/quickApp/src/main/java/com/globant/topicthree/exerciseone/Product.java

```diff
@@ -0,0 +1,30 @@
+package com.globant.topicthree.exerciseone;
+
+import java.util.UUID;
+
+/**
+ * This class represent a product, and is used to populate a shopping cart.
+ * 
+ * @author andres.vaninetti
+ *
+ */
+public class Product {
+	
+	private String id;
+	
+	/**
+	 * Constructor method
+	 * 
+	 */
+	public Product() {
+		this.id = UUID.randomUUID().toString();
```

Comment:
This won't work, everytime you create a new product it will have a new id even if its the same product. ID should be a property saved in the DB.